DP02_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<ul><li>Why is it expected? </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>This is the difference between two gaussian-like distributions that only have a small offset of the peak.</p>